### ISSUE TITLE
Separate vignette for simulations?

### ISSUE THREAD

**Issue by andrechalom** [piLaboratory/sads#114]
Should we write a separate vignette for documenting the differences in the "Connolly" bootstraping for species and individuals and the parameters of the rsad / rrad functions?

**Comment by piklprado** [comment]
Great idea. There are many points to clarify, such as : 
- those raised by issues #110, #111, #113, 
- If makes sense to sample distributions that already are deduced from samples like dpoilog, dmzsm, dvolkov or dls.
- How to perform the different bootstraping proposed by Connoly using the package
- The difference of using the `[r]` random samples from distributions and Poisson/Negative binomila sampling with rsad/rrad
- How rsad and rrad work
- ... (to be continued)

**Comment by andrechalom** [comment]
Thinking about it, maybe the best format for explaining the simulation codes is not a vignette, but a paper on arXiv, as it will allow us to show more elaborate simulation schemes without overloading the CRAN servers with repeated checks of the package vignettes.

**Comment by piklprado** [comment]
Good point. I am changing the issue title to remind us. But i think we can keep this as milestone to v.1.0.0
